www/components/Docs/Documentation.tsx

```diff
@@ -1,20 +1,20 @@
-import { Text } from "@welcome-ui/text";
+import { MarginlessText, Text } from "../core/typography/Text";
 
 interface Props {
   children: string[];
 }
 
 export default function Documentation({ children }: Props) {
-  if (children.length === 0) {
-    return <></>;
+  if (children.length > 0) {
+    return <Text>{children}</Text>;
   }
-  return <Text variant="body2">{children}</Text>;
+  return null;
 }
 
 function ShortDocumentation({ children }: Props) {
-  if (children.length === 0) {
-    return <></>;
+  if (children.length > 0) {
+    return <MarginlessText>{children[0].split("\n")[0]}</MarginlessText>;
   }
-  return <span>{children[0].split("\n")[0]}</span>;
+  return null;
```

Comment:
super nit: I generally prefer to have simple cases covered in `if`s, return early, and then the single complex case in the body. That way the simple cases pop out nicely

Reply:
Fair enough. Is it enough if I change it in my next PR? Gonna change **_all the things_** to ES6 arrow functions anyway...